torchao/prototype/moe_training/kernels/mxfp8_blocked_scales.py

```diff
@@ -256,147 +262,123 @@ def triton_scale_swizzle_per_group_2d(
         block_row_id += 1
 
 
-def triton_mx_block_rearrange_per_group_3d(
-    scales_tensor: torch.Tensor,
-) -> torch.Tensor:
+@torch.library.custom_op(
+    "torchao::triton_mx_block_rearrange_per_group_3d", mutates_args=()
+)
+def triton_mx_block_rearrange_per_group_3d(scale_tensor: torch.Tensor) -> torch.Tensor:
     """
-    Rearranges an E8M0 tensor scale to block-scaled swizzle format.
+    Rearranges an E8M0 tensor scale from row-major format to block-scaled swizzle format.
```

Comment:
Does it need to be row major? It looks like you're passing in strides, but you're also not ensuring that it is row major.

Reply:
Oh this is an outdated docstring, copied from the kernel used for dense models - where it is also outdated! Just updated it in both places, thanks.